manifests/plugin/pulp.pp

```diff
@@ -19,6 +19,8 @@
 #
 # $client_authentication:: An array of client authentication types supported by the Pulp installation.
 #
+# $rhsm_url::              The RHSM base URL represents the URL to the Katello API for RHSM traffic, or a reverse proxy to it. Katello will direct Content Hosts registering through this SmartProxy server to send subscription-manager traffic to this URL.
```

Comment:
```suggestion
# $rhsm_url::              The RHSM base URL represents the URL to the Katello API for RHSM traffic, or a reverse proxy to it. Katello will direct hosts registering through this Smart Proxy to send subscription-manager traffic to this URL.
```

Reply:
I believe Content Host is the correct terminology since all data regarding registration, subscription, errata applicability, etc. is associated with the Content Host object on Katello, while the Host object represents the infrastructure side.

Reply:
If we are getting technical, you have a Host with a subscription facet and a content facet. The idea of Content Host is more of a UI concept due to not having a unified UI. If you look at our APIs they are all rooted on the hosts API. And folks are working on unifying the UI. So, I think we should stick with the unified concept of Host.

Reply:
OK, fair point.

I was thinking from the UI perspective that the registration process happens under the Content Hosts menu: `Hosts > Content Hosts > Register Content Host`. In fact, I don't know if this was always the case but looking at Katello 4.2 I also see `Hosts > Register Host` so I can see how that will be conceptually cleaner going forward.